### DISCUSSION TITLE
Can I redirect http traffic to https

### DISCUSSION THREAD

**Discussion by lunchboxer** [beginner-corp/begin-community#76] [Support] [answered]
I have a begin app that serves static content and a graphql api at a single endpoint. Is it possible to set up http forwarding to https? Currently, I am only using the begin.app subdomains as I develop the app.  So http://subdomain-staging.begin.app gets a 404 instead of redirecting to https://subdomain-staging.begin.app which would seem to be the preferred behaviour.

**Comment by ryanblock** [comment, marked as answer]
Unfortunately no, this is a limitation of AWS API Gateway: https://stackoverflow.com/questions/47311081/redirect-http-requests-to-https-on-aws-api-gateway-using-custom-domains

Paid-tier Begin apps get this capability when behind a domain name, as we front your APIG with a CloudFront CDN distribution.

Sorry, I wish they'd support this feature!
